src/test/java/com/flowpowered/math/test/HashFunctionsTest.java

```diff
@@ -0,0 +1,156 @@
+package com.flowpowered.math;
+
+import com.flowpowered.math.HashFunctions;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class HashFunctionsTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
```

Comment:
It doesn't look like we need this, and the associated import statement for Rule, as there are no tests which throw exceptions here.